### PR TITLE
arangodb 3.7.2

### PR DESCRIPTION
**Pull request by KVS85**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Review by bayandin — CHANGES_REQUESTED** [review]
Could you please squash all your commits into single one `arangodb 3.7.2`

File: Formula/arangodb.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@ class Arangodb < Formula
   depends_on "ccache" => :build
   depends_on "cmake" => :build
   depends_on "go@1.13" => :build
+  depends_on "python@3" => :build
```
Comment (SMillerDev) [inline Formula/arangodb.rb:18]:
```suggestion
  depends_on "python@3.8" => :build
```

**Comment by KVS85** [comment]
> Could you please squash all your commits into single one `arangodb 3.7.2`

Done.

File: Formula/arangodb.rb (hunk 1 above)
Comment (bayandin) [reply Formula/arangodb.rb]: Ah, have missed this one, thanks for catching! 👍

**Review by bayandin — APPROVED** [review]

**Pull request closed without merging**